src/doc/rustc/src/platform-support.md

```diff
@@ -169,6 +169,7 @@ target | std | notes
 `i686-unknown-freebsd` | ✓ | 32-bit FreeBSD [^x86_32-floats-return-ABI]
 `i686-unknown-linux-musl` | ✓ | 32-bit Linux with musl 1.2.3 [^x86_32-floats-return-ABI]
 [`i686-unknown-uefi`](platform-support/unknown-uefi.md) | * | 32-bit UEFI
+[`loongarch64-unknown-linux-musl`](platform-support/loongarch-linux.md) | ? |  | LoongArch64 Linux (LP64D ABI) with musl 1.2.3
```

Comment:
Isn't it 1.2.5?

Reply:
Yes. The minimum version for LoongArch is 1.2.5. My initial idea was to indicate the musl version when upgrading to Tier 2, because that would be more certain: https://github.com/heiher/rust/commit/bd295974fc4fe8b9427e190a77f482d330a075e8#diff-98c317f447306855c6e723385f2a38b516acb20296d4131b5572a2ca1b112addR102

Reply:
We only build musl 1.2.3 in CI so when you want to upgrade this target to Tier 2, you'll need to figure out how to get 1.2.5 building. 

Is musl 1.2.5 the first version with LoongArch support?

Reply:
Yes.  I've figured out a way that works.  It use `crosstool-ng` to build the `loongarch64-linux-musl` cross-compilation toolchain, which includes musl 1.2.5.  Here is the patch for CI: https://github.com/heiher/rust/commit/88d8374808f578c0bae15989aaaf10a908565499